File: practice/recursive.php
function fibo($n){
	if ($n < 2) {
		return 1;
	}
	return fibo($n - 1) + fibo($n - 2);
}

echo fibo(3);
echo fibo(4);
echo fibo(5);

function facto($n){
	if ($n < 2) {
		return 1;
	}
	return $n * facto($n - 1);
}

echo facto(3);
echo facto(4);
echo facto(5);

File: practice/recursive.rb
def fibo(int)
	return 1 if int < 2 
	fibo(int - 1) + fibo(int - 2)
end

p fibo(3)
p fibo(4)
p fibo(5)

def factorial(int)
	return 1 if int < 2
	int * factorial(int - 1)
end

p factorial(3)
p factorial(4)
p factorial(5)
